templates/feature.html

```diff
@@ -16,6 +16,10 @@
 <link rel="alternate" type="application/rss+xml" href="http://www.chromestatus.com/features.xml" title="All features" />
 {% endblock %}
 
+{% block html_imports %}
+<link rel="import" href="/static/elements/features-imports{% if VULCANIZE %}.vulcanize{% endif %}.html">
```

Comment:
In theory, at some point we should be able to just switch this page over to use `<chromedash-feature>`.